### ISSUE TITLE
libsigc++-2.10.8 requires docbook stylesheet for building the documentation

### ISSUE THREAD

**Issue by bedroge** [easybuilders/easybuild-easyconfigs#18292]
I ran into the following when building `libsigc++-2.10.8-GCCcore-10.3.0.eb` in a quite minimal Rocky 8 container:
```
make[2]: Entering directory '/dev/shm/p251204/libsigc/2.10.8/GCCcore-10.3.0/libsigc++-2.10.8/docs'
  GEN      reference/libsigc++-2.0.tag
  GEN      manual/html/index.html
warning: Tag 'CLANG_ASSISTED_PARSING' at line 144 of file 'reference/Doxyfile' belongs to an option that was not enabled at compile time.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u", or recompile doxygen with this feature enabled.
warning: Tag 'CLANG_OPTIONS' at line 145 of file 'reference/Doxyfile' belongs to an option that was not enabled at compile time.
         To avoid this warning please remove this line from your configuration file or upgrade it using "doxygen -u", or recompile doxygen with this feature enabled.
Variable $SGML_CATALOG_FILES not set
error : Unknown IO error
warning: failed to load external entity "http://docbook.sourceforge.net/release/xsl/current/html/chunk.xsl"
cannot parse http://docbook.sourceforge.net/release/xsl/current/html/chunk.xsl
```

This looked similar to https://github.com/easybuilders/easybuild-easyconfigs/issues/8893, so I tried adding `docbook-style-xsl` to the container, which solved the issue. Disabling the documentation, as was done for #8893 in #13410, is also possible by adding `configopts = "--disable-documentation"`. I'm not sure if `libsigc++ 3` has the same issue.

**Comment by branfosj** [comment]
My vote goes to disabling the documentation.

**Comment by bedroge** [comment]
Version 3 seems to build fine without that package, perhaps because of the switch to CMake.

I've opened PR #18301 to fix this for all 2.10.x versions by disabling the documentation.
